test/functional/cursor.test.js

```diff
@@ -1550,6 +1550,50 @@ describe('Cursor', function () {
     }
   });
 
+  it('does not auto destroy streams', {
+    metadata: {
+      requires: { topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'] }
+    },
+
+    test: function (done) {
+      const docs = [];
+
+      for (var i = 0; i < 10; i++) {
+        docs.push({ a: i + 1 });
+      }
+
+      const configuration = this.configuration;
+      const client = configuration.newClient(configuration.writeConcernMax(), { maxPoolSize: 1 });
+      client.connect((err, client) => {
+        expect(err).to.not.exist;
+        //this.defer(() => client.close());
```

Comment:
cruft? I see the close in the end handler below if that was intended.

Reply:
Updated.